label/reports/dynamic_label.xml

```diff
@@ -0,0 +1,80 @@
+<openerp>
+  <data>
+    <template id="report_label">
+      <style type="text/css">
+	div.o_label_div {
```

Comment:
Please use more descriptive names for css classes (e.g. o_label_page, o_label_content, etc).